salary_prediction_with_linear_regression/salary_prediction_linear_regression.cc

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file salary_prediction_linear_regression.cc
+ *
+ * A simple example usage of Linear Regression
+ * applied to Salary dataset
+ */
+#include<mlpack/core.hpp>
+#include<mlpack/core/data/split_data.hpp>
+#include<mlpack/methods/linear_regression/linear_regression.hpp>
+#include<cmath>
```

Comment:
```suggestion
#include <mlpack/core.hpp>
#include <mlpack/core/data/split_data.hpp>
#include <mlpack/methods/linear_regression/linear_regression.hpp>
#include <cmath>
```
Just a simple style sugestion---I think maybe it would be worth going through the file for style changes.